src/Task/Codeception.php

```diff
@@ -60,6 +60,8 @@ public function run(ContextInterface $context): TaskResultInterface
         $arguments->add('run');
         $arguments->addOptionalArgument('--config=%s', $config['config_file']);
         $arguments->addOptionalArgument('--fail-fast', $config['fail_fast']);
+        $arguments->addOptionalArgument('--xml', $config['xml']);
```

Comment:
You need to configure these available options in `getConfigurableOptions()` as well.

Reply:
done!